test/smoke/framework/testCore/src/main/java/com/microsoft/applicationinsights/smoketest/AiSmokeTest.java

```diff
@@ -225,19 +235,38 @@ public void run() {
 	}
 
 	@ClassRule
-	public static TestWatcher agentModeLoader = new TestWatcher() {
+	public static TestWatcher classOptionsLoader = new TestWatcher() {
 		@Override
 		protected void starting(Description description) {
+			System.out.println("Configuring test class...");
 			UseAgent ua = description.getAnnotation(UseAgent.class);
 			if (ua != null) {
 				agentMode = ua.value();
 				System.out.println("AGENT MODE: "+agentMode);
 			}
+			WithDependencyContainers wdc = description.getAnnotation(WithDependencyContainers.class);
+			if (wdc != null) {
+				for (DependencyContainer container : wdc.value()) {
+					if (container.value() == null || container.value().trim().isEmpty()) {
```

Comment:
I would prefer removing the long if check with isNotBlank() method from apache-commons-lang. I think we already use it in our project.

https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isNotBlank(java.lang.String)

Reply:
thanks for the tip.